data/billing.py

```diff
@@ -348,7 +348,8 @@
     },
 ]
 
-RH_SKUS = [plan["rh_sku"] for plan in PLANS if plan.get("rh_sku") is not None]
+RH_SKUS = ["MW02701"]
+RECONCILER_SKUS = [plan["rh_sku"] for plan in PLANS if plan.get("rh_sku") is not None]
```

Comment:
Do these need to be separate?

Reply:
Yeah, because the old skus will be used to track stripe customers in the RH portal, and the new skus will be what is used for binding to orgs, etc.